test/preview/components/test_prompt.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+
+import haystack
+from haystack.preview.components.prompt.prompt import Prompt
```

Comment:
`from ... prompt.prompt import Prompt` isn't ideal but let's not focus on naming aspects in this PR. `from ... components import Prompt` should suffice her though, no?

Reply:
Sure no problem :+1: